src/share/classes/com/sun/javatest/regtest/agent/JUnitRunner.java

```diff
@@ -133,6 +135,7 @@ private static void runWithJUnitPlatform(Class<?> mainClass) throws Exception {
             SummaryGeneratingListener summaryGeneratingListener = new SummaryGeneratingListener();
 
             AgentVerbose verbose = AgentVerbose.ofStringRepresentation(System.getProperty("test.verbose"));
+            Logger.getLogger("org.junit").setLevel(Level.WARNING);
```

Comment:
Looks OK to me. I don't think we use/rely on org.junit logs in any parts of jtreg test execution or its reports. So I think it's OK to set this level to warning to prevent the noisy:

```
INFO: Skipping cleanup of temp dir JTwork/scratch/junit-7718269284146213374 for parameter 'arg0' in method fooBar(Path) due to CleanupMode.NEVER.
```

Reply:
Note that users may opt-in into more detail messages from JUnit. For example, like so:

```java
    @BeforeAll
    static void setup() {
        Logger.getLogger("org.junit.jupiter.engine.extension.TempDirectory$CloseablePath")
                .setLevel(Level.ALL);
    }
```